### PR TITLE
Fabo/2512 inclusion step

### LINKED ISSUE #2512: show "waiting for Tx to be included in block" as step in action modal transaction flow
**Is your feature request related to a problem? Please describe.**

Currently when you send a tx, on the signing step it will wait and show "Waiting for Ledger" until the tx is included. We should show another step "Waiting for Tx to be included". Where we show a message "Your transaction has been send.  Waiting for the transaction to be included in a block.".
If the transaction times out, add a description: "Your transaction wasn't included in a block until now. It still could be included later."
Bonus: Show the amounts of blocks the user has already waited.
Maybe increase the waiting time for txs as well. It already timed out for me (for 0 fees)

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by faboweb**
Closes #2512 

**Description:**

image

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

File: src/ActionModal/components/ActionModal.vue [hunk 1]
```diff
@@ -139,6 +142,19 @@
...
         </form>
       </div>
+      <div v-else-if="step === `send`" class="action-modal-form">
+        <TmDataMsg icon="hourglass_empty">
+          <div slot="title">
+            Sent and confirming
+          </div>
+          <div slot="subtitle">
+            The transaction
+            <!--with the hash {{ txHash }}-->
```
Comment (colw) [inline src/ActionModal/components/ActionModal.vue]: Is this meant to be commented still?

Comment (faboweb) [reply src/ActionModal/components/ActionModal.vue]: Left in there to signal possible improvements. I can remove it.

Comment (colw) [reply src/ActionModal/components/ActionModal.vue]: It's fine 👍

**Comment by colw** [comment]
Also, fixes #2788 ?

**Comment by faboweb** [comment]
> Also, fixes #2788 ?

Nop.

File: src/ActionModal/components/ActionModal.vue [hunk 2]
```diff
@@ -12,173 +12,157 @@
...
-          :bond-denom="bondDenom"
-        />
-        <TmFormMsg
-          v-if="$v.invoiceTotal.$invalid"
-          name="Total"
-          type="between"
-          min="0"
-          :max="balanceInAtoms"
+        <Steps
+          :steps="['Details', 'Fees', 'Sign', 'Send']"
```
Comment (jbibla) [inline src/ActionModal/components/ActionModal.vue]: i think we need a solution that doesn't have 4 steps:

1. user entered *details*
2. show LiTx above *fees*
3. approve (this is slightly different for ledger + extension)
3a. detect and sign on external device
3b. waiting for inclusion
3c. success / failure (retry)

Comment (faboweb) [reply src/ActionModal/components/ActionModal.vue]: Having 3 subpoints for 3. hits my alarm

**Pull request merged**